wordpress-mobile/WordPress-Android
# Pulling avoid-jitpack changes into gutenberg release branch

Pulling changes to build gutenberg-mobile locally without JitPack from the release/14.8 branch into the `gutenberg/integrate_release_1.28.0` so that branch can successfully build on CI.

To test:

Build debug and release builds locally with `BUILD_GUTENBERG_FROM_SOURCE` set to `false`. Also verify that debug builds can be built with `BUILD_GUTENBERG_FROM_SOURCE` set to `true` (so 3 combinations total).

PR submission checklist:
